myst_nbs/time_series/bayesian_var_model.myst.md

```diff
@@ -0,0 +1,751 @@
+        shade_background(ppc, ax, i, "inferno")
+    ax[0].plot(np.arange(ppc.shape[0]), ppc[:, 0, :].mean(axis=1), color="cyan", label="Mean")
+    ax[0].plot(temp["dl_gdp"], "o", color="white", markersize=6, label="Observed")
+    ax[0].plot(temp["dl_gdp"], "o", color="black", markersize=4, label="Observed")
+    ax[0].set_title(f"Posterior Predictive GDP: {country}")
+    ax[0].legend(loc="lower left")
+    ax[1].plot(temp["dl_cons"], "o", color="white", markersize=6, label="Observed")
+    ax[1].plot(temp["dl_cons"], "o", color="black", markersize=4, label="Observed")
+    ax[1].plot(np.arange(ppc.shape[0]), ppc[:, 1, :].mean(axis=1), color="cyan", label="Mean")
+    ax[1].set_title(f"Posterior Predictive Consumption: {country}")
+    ax[1].legend(loc="lower left")
+    ax[2].plot(temp["dl_gfcf"], "o", color="white", markersize=6, label="Observed")
+    ax[2].plot(temp["dl_gfcf"], "o", color="black", markersize=4, label="Observed")
+    ax[2].plot(np.arange(ppc.shape[0]), ppc[:, 2, :].mean(axis=1), color="cyan", label="Mean")
+    ax[2].set_title(f"Posterior Predictive Investment: {country}")
+    ax[2].legend(loc="lower left")
+plt.suptitle("Posterior Predictive Checks on Hierarchical VAR", fontsize=20);
+```
+
+Here we can see that the model appears to have recovered reasonable posterior predictions for the observed data and the volatility of the Irish GDP figures is clear next to the other countries. Whether this is a cautionary tale about data quality or the corruption of metrics we leave to the economists to figure out.
+
++++
+
+## Conclusion
+
+VAR modelling is a rich an interesting area of research within economics and there are a range of challenges and pitfalls which come with the interpretation and understanding of these models. We hope this example encourages you to continue exploring the potential of this kind of VAR modelling in the Bayesian framework. Whether you're interested in the relationship between grand economic theory or simpler questions about the impact of poor app performance on customer feedback, VAR models give you a powerful tool for interrogating these relationships over time. As we've seen Hierarchical VARs further enables the precise quantification of outliers within a cohort and does not throw away the information because of odd accounting practices engendered by international capitalism. 
+
+In the next post in this series we will spend some time digging into the implied relationships between the timeseries which result from fitting our VAR models.
+
++++
+
+## References
+
+:::{bibliography}
+:filter: docname in docnames
+:::
+
++++
+
+## Authors
+* Adapted from the PYMC labs [Blog post](https://www.pymc-labs.io/blog-posts/bayesian-vector-autoregression/) and Jim Savage's discussion [here](https://rpubs.com/jimsavage/hierarchical_var) by [Nathaniel Forde](https://nathanielf.github.io/) in November 2022 ([pymc-examples#456](https://github.com/pymc-devs/pymc-examples/pull/456))
+
+```{code-cell} ipython3
```

Comment:
can you also add a heading saying watermark? https://docs.pymc.io/en/latest/contributing/jupyter_style.html#watermark

Reply:
Done.